libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -107,23 +107,23 @@ const AP_Param::GroupInfo AC_AttitudeControl_Multi::var_info[] = {
     // @Param: RAT_PIT_FLTT
     // @DisplayName: Pitch axis rate controller target frequency in Hz
     // @Description: Pitch axis rate controller target frequency in Hz
-    // @Range: 1 50
+    // @Range: 5 50
```

Comment:
FWIW I think this should be 100

Reply:
@lthall, what do you think about making the max for ATC_RAT_PIT_FLTT and ATC_RAT_RLL_FLTT 100 instead of 50?

Reply:
ok, updated to 100, txs